html-css/week1/Ali-Nematollahi/index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ali-Nematollahi-CV</title>
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0&icon_names=calendar_month,language,location_on,menu,schedule" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <div id="container">
+        <nav>
+            <ul>
+                <li>
+                    <a href="#">SUMMARY</a>
+                </li>
+                <li>
+                    <a href="#">SKILLS</a>
+                </li>
+                <li>
+                    <a href="#">PORTFOLIO</a>
+                </li>
+                <li>
+                    <a href="#">CONTACT</a>
+                </li>
+            </ul>
+        </nav>       
+            <header>
+                <div id="name">
+                    Ali Nematollahi
+                </div>
+                <div id="title">
+                    Full-Stack Developer
+                </div>
```

Comment:
Div tags don't hold any semantic meaning. They just wrap content. Instead, you could use one of the H1 to H6 headlines.
For example:
```suggestion
                <h1 class="header__title">Ali Nematollahi </h1>
                <p class="header__description">Full-Stack Developer</p>
```